lib/getStepDefinitionsPaths.js

```diff
@@ -1,18 +1,25 @@
 const glob = require("glob");
 const cosmiconfig = require("cosmiconfig");
+const process = require("process");
 const stepDefinitionPath = require("./stepDefinitionPath.js");
 const { getStepDefinitionPathsFrom } = require("./getStepDefinitionPathsFrom");
 
 const getStepDefinitionsPaths = filePath => {
+  const appRoot = process.cwd();
   let paths = [];
   const explorer = cosmiconfig("cypress-cucumber-preprocessor", { sync: true });
   const loaded = explorer.load();
   if (loaded && loaded.config && loaded.config.nonGlobalStepDefinitions) {
     const nonGlobalPattern = `${getStepDefinitionPathsFrom(
       filePath
     )}/**/*.+(js|ts)`;
-    const commonPath =
-      loaded.config.commonPath || `${stepDefinitionPath()}/common/`;
+
+    let commonPath = `${stepDefinitionPath()}/common/`;
+
+    if (loaded.config.commonPath) {
+      commonPath = `${appRoot}/${loaded.config.commonPath}`;
```

Comment:
I think this should be also `path.normalize()` to work in Windows (same with rest of paths)